fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/qrcode/QrScannerFragment.java

```diff
@@ -66,7 +61,8 @@ public View onCreateView(
     scanningViewModel =
         scanningAction.obtainScannerViewModel(requireActivity(), popViewModel.getLaoId());
 
-    if (scanningAction != ScanningAction.ADD_LAO_PARTICIPANT) {
+    if (scanningAction != ScanningAction.ADD_LAO_PARTICIPANT
+        && scanningAction != ScanningAction.ADD_POPCHA) {
```

Comment:
would it make sense to have `displayCounter` in the fragment bundle ? 
Basically, should this corner case be handled by a long series of `if not`, or can we just let the application calling the fragment decide if they want a counter or not ?

Reply:
Yes, that is a nice choice design-wise, I'll add it